test/tests/nek_errors/deformation/box-test1.i

```diff
@@ -0,0 +1,149 @@
+[GlobalParams]
+  use_displaced_mesh = true
+  displacements = 'disp_x_o disp_y_o disp_z_o'
+[]
+
+[Mesh]
+  type = FileMesh
+  file = box.msh
+  parallel_type = replicated
+  order = SECOND
+  displacements = 'disp_x_o disp_y_o disp_z_o'
```

Comment:
So, from my understanding, everything put into `[GlobalParams]` essentially gets copied into all the other blocks. So I don't think we need the separate `[GlobalParams]` block?

Reply:
I'll test it and remove it from the `[Mesh]` block. I think you asked a question about this in the MOOSE slack a few months ago, and someone responded that `displacements` and `use_displaced_mesh` in `[GlobalParams]` don't always get applied to other blocks. Is that still true? If so, maybe we should avoid `[GlobalParams]` altogether from now on.

Reply:
Ohhh I totally forgot about that. I think you're right - if we _can_ avoid `[GlobalParams]`, then let's do that. Can you test it out and see what the behavior is to remind us?

Reply:
I think it works either way for this case, but I have had experiences in the past where it triggers MOOSE errors when using the Tensor Mechanics module.